tests/js/spec/stores/selectedGroupStore.spec.jsx

```diff
@@ -211,4 +211,61 @@ describe('SelectedGroupStore', function () {
       expect(trigger).toHaveBeenCalled();
     });
   });
+
+  describe('shiftSelectItems()', function () {
```

Comment:
Do you think it might be good to have RTL tests for this behavior? Or is that not an easy ask with the store being involved

Reply:
@malwilley its not a great option right now since StreamGroup is mocked out in the overview tests. It would be nice though